atlasdb-client/src/main/java/com/palantir/atlasdb/util/TaggedMetricsInvocationEventHandler.java

```diff
@@ -176,18 +176,13 @@ public void onFailure(@Nullable InvocationContext context, @Nonnull Throwable ca
                         .safeTags(tags)
                         .build())
                 .mark();
-        taggedMetricRegistry
-                .meter(MetricName.builder()
-                        .safeName(MetricRegistry.name(
-                                failuresMetricName, cause.getClass().getName()))
-                        .safeTags(tags)
-                        .build())
-                .mark();
-    }
-
-    private void markGlobalFailure() {
```

Comment:
for this: yeah I found https://github.com/palantir/atlasdb/pull/3075, I'm not sure that has more context on why we did track global failures in the past though. I don't recall having used it recently at all.